acct_mgt/moc_openshift.py

```diff
@@ -355,18 +355,18 @@ def project_exists(self, project_name):
             return True
         return False
 
-    def create_project(self, project_name, display_name, user_name):
+    def create_project(self, project_name, display_name, user_name, annotations={}):
```

Comment:
Never use mutable objects in function definitions in Python.

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

(Random article I quickly found with a funny title)

Reply:
fixed